katello/katello-change-hostname

```diff
@@ -217,6 +217,10 @@ STDOUT.puts installer_output
 STDOUT.puts "Regenerating ueber certs"
 `foreman-rake katello:regenerate_ueber_certs`
 
+STDOUT.puts "Restarting puppet services"
+`service puppet restart`
+`service puppetserver restart`
```

Comment:
systemctl ? were not supporting el6 now ;) 
Also is puppetserver running on every box now? I though we run it with Passenger, at least of Puppet 3